concourse/verify-production-urls/script.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -ex
+
+bucket=sandbox-unified-web-primary
```

Comment:
this can be pulled in from a param, params are used like this https://github.com/openstax/unified-deployment/blob/master/concourse/update-release-candidate-environments/task.yml and the s3 bucket is param is called `prod-unified-s3-bucket`